templates/taxi/car_detail.html

```diff
@@ -7,6 +7,8 @@ <h1>Drivers</h1>
   <ul>
   {% for driver in car.drivers.all %}
     <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
+    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-warning">Update car</a>
+    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-warning">Delete</a>
```

Comment:
The 'Update car' and 'Delete' buttons are placed inside the loop iterating over drivers. This will cause the buttons to be repeated for each driver. Consider moving these buttons outside the loop to display them only once.